polaris.shopify.com/src/components/Card/Card.module.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../styles/mixins.scss';
+.Card {
+  @include responsive-props(
+    'card',
+    'padding-block-start',
+    'padding-block-start'
+  );
+  @include responsive-props('card', 'padding-block-end', 'padding-block-end');
+  @include responsive-props(
+    'card',
+    'padding-inline-start',
+    'padding-inline-start'
+  );
+  @include responsive-props('card', 'padding-inline-end', 'padding-inline-end');
+
+  border-radius: var(--border-radius-700);
+  background-color: var(--surface);
+  border-radius: var(----p-border-radius-3, 12px);
+  border-radius: var(----p-border-radius-3, 12px);
+
+  // As this will be used with Markdown elements and components
+  // We want to preserve margin rules for all children
+  // Except the last-child, so as not to introduce unecessary space
+  // at the bottom of the card
+  > :last-child {
+    margin-bottom: 0;
+  }
+
+  box-shadow: 0px 1px 0px 0px rgba(26, 26, 26, 0.07),
```

Comment:
these values are taken straight from Figma, I'm unclear if these should replace the existing [--card-shadow token](https://github.com/Shopify/polaris/blob/040740989250a57889a16a22f599fb7452cfe68a/polaris.shopify.com/src/styles/globals.scss#L208), I've left it localised for now.

Reply:
i think this is fine for now, we can update this value once we've finalized token value updates